novobi_henry_book/controllers/add_new_book.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import json
+from odoo import http
+from odoo.http import request
+
+
+class AddBookController(http.Controller):
+    @http.route(['/library_book/add_new_book'], type='http', auth="public",csrf=False,methods=["POST"])
+    def create_new_book(self, **kw):
+        model_name = "library.book"
+        in_status = ["not published", "available", "borrowed", "lost"]
+
+        if kw:
+            ISBN = kw['isbn']
+            name = kw['name']
+            short_name = kw['short_name']
+            status = kw['status']
+            if(status not in in_status):
+                status = "available"
+
+        isbn = request.env[model_name].sudo().search([("ISBN", "=", ISBN)])
```

Comment:
We should declare onetime for sale request environment: 
For example: LibraryBook = request.env['library.book'].sudo()